docs/docker-stack/docker-examples/restricted/restricted_environments.sh

```diff
@@ -22,28 +22,29 @@ AIRFLOW_SOURCES="$(cd "$(dirname "${BASH_SOURCE[0]}")/../../../../" && pwd)"
 cd "${AIRFLOW_SOURCES}"
 
 # [START download]
+export AIRFLOW_VERSION="2.3.0.dev0"
 rm docker-context-files/*.whl docker-context-files/*.tar.gz docker-context-files/*.txt || true
 
 curl -Lo "docker-context-files/constraints-3.7.txt" \
-    https://raw.githubusercontent.com/apache/airflow/constraints-2.3.0.dev0/constraints-3.7.txt
+    "https://raw.githubusercontent.com/apache/airflow/constraints-${AIRFLOW_VERSION}/constraints-3.7.txt"
 
 # For Airflow pre 2.1 you need to use PIP 20.2.4 to install/download Airflow packages.
 pip install pip==20.2.4
 
 pip download --dest docker-context-files \
     --constraint docker-context-files/constraints-3.7.txt  \
-    "apache-airflow[async,aws,azure,celery,dask,elasticsearch,gcp,kubernetes,postgres,redis,slack,ssh,statsd,virtualenv]==2.0.2"
+    "apache-airflow[async,aws,azure,celery,dask,elasticsearch,gcp,kubernetes,postgres,redis,slack,ssh,statsd,virtualenv]==${AIRFLOW_VERSION}"
```

Comment:
Was this just a typo? The version is `2.3.0.dev0` everywhere else in this file.

Reply:
Should be 2.2. in both cases. It was originally 2.0.2 in both cases, but has been replaced by one of bulk version replacements apparently.

Reply:
This field is updated to version according to `setup.py`. https://github.com/apache/airflow/blob/aff647715baca7da26edf635abd42c7e911f5313/scripts/ci/pre_commit/pre_commit_update_versions.py#L24-L29
I can change it, but it will cause a lot of unrelated changes.

Reply:
I just removed the file from auto-fix (see the fixup i added - if you are ok we can merge it) . The cases with customization can be easily updated manually (and they do not even have to be because they are about using specific version, so it does not really matter which version is used).

Reply:
I also updated other versions - If you want you can remove my fixup and I can make a separate PR out of that - BTW I just run all the "customize" image scripts and they are working fine .

Reply:
@potiuk LGTM